Comanche055/TVCRESTARTS.agc

```diff
@@ -34,47 +34,47 @@
 
 # Page 956
 # NAME....TVCRESTART PACKAGE, CONSISTING OF REDOTVC, ENABL1, 2, CMDSOUT, PHSCHK2, ETC.
-# LOG SECTION....TVCRESTARTS			SUBROUTINE....DAPCSM
+# LOG SECTION...TVCRESTARTS			SUBROUTINE...DAPCSM
 # MODIFIED BY SCHLUNDT				21 OCTOBER 1968
 #
 # FUNCTIONAL DESCRIPTION....
 #
 #      *RESTART-PROCESS THE TVC DAPS, INCLUDING PITCHDAP, YAWDAP,
```

Comment:
Should be `*RESTART-PROOFS`. Technically, these bullet points should go into the indentation, not beside it (see [scan](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0956.jpg)).